manifests/g/Git/Git/2.41.0/Git.Git.installer.yaml

```diff
@@ -15,17 +15,23 @@ Commands:
 - git
 ReleaseDate: 2023-06-01
 InstallerSwitches:
-  Custom: /SUPPRESSMSGBOXES
+  Custom: /SUPPRESSMSGBOXES /enable-component windowsterminal
 Installers:
 - Architecture: x64
   Scope: machine
   InstallerUrl: https://github.com/git-for-windows/git/releases/download/v2.41.0.windows.1/Git-2.41.0-64-bit.exe
   InstallerSha256: 45dc30410916b8ec5501be39d01d5b60535731c04fa68283b4f9df4920877d4e
   ElevationRequirement: elevatesSelf
+  Dependencies:
+    PackageDependencies:
+    - PackageIdentifier: Microsoft.WindowsTerminal
```

Comment:
This is incorrect. `/enable-component windowsterminal` does not introduce dependency on Windows Terminal.
The registration is safe to do even if WT is not installed: https://github.com/microsoft/terminal/issues/15559#issuecomment-1593684739